src/core/components/hv-route/index.tsx

```diff
@@ -628,9 +628,23 @@ function HvRouteFC(props: Types.Props) {
 
   return (
     <HvRouteInner
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...{ ...props, ...navigationContext, ...navigatorMapContext }}
```

Comment:
The order in which the final object might be changing - we used to have
`props`, on top of which we merged `navigationContext`, then `navigatorMapContext` (in this order).
Is there any possible overlap of props between these 3 objects, and if so, does this refactor accounts for it?

Reply:
Confirmed 'url' is the only overlapping property and is handled properly